test/e2e/features/retries.go

```diff
@@ -33,50 +33,100 @@ var _ = ginkgo.Describe("retries", func() {
 		APISIXRouteVersion:    "apisix.apache.org/v2alpha1",
 	}
 	s := scaffold.NewScaffold(opts)
-	ginkgo.It("active check", func() {
+
+	routeTpl := `
+apiVersion: apisix.apache.org/v2alpha1
+kind: ApisixRoute
+metadata:
+	name: httpbin-route
+spec:
+	http:
+	- name: rule1
+	match:
+		hosts:
+		- httpbin.org
+		paths:
+		- /*
+	backend:
+		serviceName: %s
+		servicePort: %d
+`
+	ginkgo.It("is missing", func() {
 		backendSvc, backendPorts := s.DefaultHTTPBackend()
+		ar := fmt.Sprintf(routeTpl, backendSvc, backendPorts[0])
+		err := s.CreateResourceFromString(ar)
+		assert.Nil(ginkgo.GinkgoT(), err)
+		time.Sleep(5 * time.Second)
 
 		au := fmt.Sprintf(`
 apiVersion: apisix.apache.org/v1
 kind: ApisixUpstream
 metadata:
   name: %s
 spec:
-  retries: 3
```

Comment:
we can't using "nil" in `ApisixUpstream.spec`

Reply:
How about change spec to 
```yaml
retries:
  type: integer
  minimum: 0
  nullable: true
```

Reply:
ok